### ISSUE TITLE
[V6]The page always refresh in IE9

### ISSUE THREAD

**Issue by raymondsze** [erikras/redux-form#1302]
Simple to reproduce
```
import React, { Component, PropTypes } from 'react';
import { Field, reduxForm } from 'redux-form';
@reduxForm({ form: 'login' })
class Form extends Component {
  renderInput() {
    return (
      <Field
        name="test"
        component="input"
        type="text"
      />
    );
  }
  render() {
    return (
      <form ref="form">
        {this.renderInput()}
      </form>
    );
  }  
}  
export default Form;
```

No error, but the page auto refresh infinitely.

I using Microsoft Edge with following simulation
Document Mode: 9
User Agent: IE9

**Comment by babsonmatt** [comment]
can you make a plunker (https://plnkr.co/) reproducing?

**Comment by raymondsze** [comment]
Sorry, I found the root cause right now, not related to redux-form, but browser history that IE9 do not support.
I am using react-router-redux to sync the history to redux store, seems when I go to the redux form, it did something initiation that update the redux store.
As history is also located in redux store, seems somehow the history operation fallback to refresh.
At least I removed the syncHistoryWithStore(browserHistory, store) in other file and it works.
I will close this issue.
